ui/src/components/LandingPackageSIzes.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useState, useEffect } from "react";
```

Comment:
There is a typo in this file name.

Reply:
File name still didn't change. It seems that git doesn't recognize file name changes if only letter cases change. You can look [here](https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git) how to fix this issue.